app/controllers/CalculationController.scala

```diff
@@ -65,7 +65,8 @@ class CalculationController @Inject()(viewLPP: CalculationLPPView, viewAdd: Calc
               }
             )
           } else {
-            Ok(viewAdd())
+            val additionalPenaltyRate = parseBigDecimalToFriendlyValue(penalty.get.financial.percentageOfOutstandingAmtCharged.getOrElse(0))
```

Comment:
Not sure if we should default the additional penalty rate - to be honest, I'm also not sure if we should use the payload as we know it's going to be 4%?
Whichever way you do it, I'd say if the value was None then we should show an error.

Reply:
Ok, in that case adding the 'additional penalty rate' as a static value of 4